infra/chart/Chart.yaml

```diff
@@ -1,4 +1,4 @@
 apiVersion: v1
 description: "Turing: ML Experimentation System"
 name: turing
-version: 0.1.1
+version: 0.1.2
```

Comment:
@ashwinath why is the chart version being bumped up? I don't think it is needed since the chart folder is not modified otherwise.

Reply:
alright reverting.